qa/L0_request_cancellation/scheduler_test.py

```diff
@@ -228,6 +243,15 @@ def test_scheduler_streaming_request_cancellation(self):
         time.sleep(2)  # ensure reaper thread has responded
         self._assert_streaming_response_is_cancelled(response)
 
+    def test_zerror_metrics(self):
```

Comment:
typo?
```suggestion
    def test_error_metrics(self):
```

Reply:
adding the 'z' ensures that the test gets run at the very end.
I want to make sure that metrics test runs at the end to leverage existing tests as much as possible.
Did not find a better way as 
"The order in which the various tests will be run is determined by sorting the test method names with respect to the built-in ordering for strings." As per python unit test docs.

Reply:
Instead of relying on test order which may be prone to subtle failures in the future, can you instead make the metrics checks more relative?

Ex: instead of checking the metrics real some absolute value like 4, you would capture them at the start of the test case as a relative baseline X, and compare the new metric value afterwards to say X+4. This should be more stable regardless of the order the tests get run.

Let me know if that makes sense.